docs/Build-for-RPi2.md

```diff
@@ -3,10 +3,11 @@
 ### Prerequisite
 #### Linux
 
-Install arm linux cross compiler.
+Install arm linux cross compiler and i2c library.
 
 ``` bash
-sudo apt-get install gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf
+sudo apt-get install gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf i2c-tools libi2c-dev
+sudo mv /usr/arm-linux-gnueabihf/include/linux/i2c-dev.h /usr/arm-linux-gnueabihf/include/linux/i2c-dev.h.kernel
```

Comment:
In addition, copying 'i2c-dev.h' file to IoT.js project will solve simply this problem, but its license is GPL. That's why I can't copy it.